src/framework/components/anim/binder.js

```diff
@@ -91,31 +91,32 @@ Object.assign(AnimComponentBinder.prototype, {
         return setter.bind(this);
     },
     _colorSetter: function (propertyComponent, propertyHierarchy) {
-        var colorKeys = ['r', 'g', 'b', 'a'];
+        var color = this._getProperty(propertyComponent, propertyHierarchy);
```

Comment:
might be worth also caching object and property in the float and boolean case?

Reply:
👍  I've went and updated the boolean / float setters.